homeassistant/components/caldav/calendar.py

```diff
@@ -161,6 +161,9 @@ async def async_get_events(self, hass, start_date, end_date):
         )
         event_list = []
         for event in vevent_list:
+            if not hasattr(event.instance, "vevent"):
+                _LOGGER.warning("Skipped event with missing 'vevent' property")
+                continue
             vevent = event.instance.vevent
```

Comment:
We have this access in the `update` method too. The stack trace in the linked issue is from that place. Don't we need to fix that too?

Reply:
Good catch, fixed :+1: